foundry.toml

```diff
@@ -10,9 +10,9 @@ remappings = [
 "eth-gas-reporter/=node_modules/eth-gas-reporter/",
 "forge-std/=lib/forge-std/src/",
 "hardhat/=node_modules/hardhat/",
-"solady/=node_modules/solady/",
-"@openzeppelin/contracts-upgradeable/=/node_modules/@openzeppelin/contracts-upgradeable/",
-"@openzeppelin/contracts/=/node_modules/@openzeppelin/contracts/",
+"solady/=node_modules/solady/src",
```

Comment:
why are we changing this?

Reply:
Foundry was having issues finding the source code for this dependency. It seems like this was because `solady`'s internal file structure changed in node modules, so I just edited the remapping to point to the src directory.